firstvoices/firstvoices/settings.py

```diff
@@ -141,8 +144,8 @@
         "loggers": {
             ELASTICSEARCH_LOGGER: {
                 "handlers": ["console"],
-                "level": "INFO",  # Change level to INFO to view connection requests
-            }
+                "level": "ERROR",  # Change level to INFO to view connection requests
```

Comment:
Any reason for this logging change? Error seems excessively strict

Reply:
I'll double check with Sahil to see if we would like to keep it at info

Reply:
"info" works as well. I initially kept it at error to prevent the logs being filled up by ES requests.